requirements/edx/base.txt

```diff
@@ -16,7 +16,7 @@ dealer==2.0.4
 defusedxml==0.4.1
 django-babel-underscore==0.5.1
 markey==0.8  # From django-babel-underscore
-django-celery==3.1.16
+django-celery==3.2.1
```

Comment:
Are the tables used / referenced on prod?  Meaning, if we trashed them in advanced manually would that help?  I assume since you mention devstack that something migrates them in during clean builds.  We'll also have to think about this during the Gingko release.